css/types/calc.json

```diff
@@ -84,11 +84,11 @@
             },
             "webview_android": [
               {
-                "version_added": "37"
+                "version_added": "4.4"
```

Comment:
The table has 4.4.3 as the earliest version and not 4.4. Or maybe make this "≤37"?

Reply:
Set to ≤37!